k8s/openebs-servicemonitor.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    app: openebs
+  name: openebs
+  namespace: openebs
+spec:
+  endpoints:
+  - path: /metrics
+    port: exporter
+  namespaceSelector:
+    matchNames:
+    - openebs
+  selector:
+    matchLabels:
+      openebs.io/cas-type: cstor
```

Comment:
Assuming that all the workloads created by openebs that export metrics have this label, if not let me know what this should be?

Reply:
@surajssd you can replace the label `openebs.io/cas-type: cstor` to `monitoring: volume_exporter_prometheus`

Reply:
This is used across all the storage-engine

Reply:
@utkarshmani1997 will it be possible to keep/include the metrics as suggested by @surajssd

Reply:
Yes I am confused if I have seen `monitoring: volume_exporter_prometheus` label on these metrics.

Reply:
Does the matchLabels .. work with -- exists operator? 

The current labels are for each volume are:
https://github.com/openebs/maya/blob/master/pkg/install/v1alpha1/cstor_volume.go#L312
https://github.com/openebs/maya/blob/master/pkg/install/v1alpha1/jiva_volume.go#L563

Currently, there is no common label - but we could add a common label across all the openebs components that support metrics by possibly doing the following:
(a) Add a generic label `openebs.io/monitoring : enabled`
(b) Create a service with named port called `exporter`

For this PR, for 0.8 version, let us mention that users may have to create two ServiceMonitor objects - one for cstor and jiva?

Reply:
Is `monitoring: volume_exporter_prometheus` common label against all types of pv providers?

Reply:
I can just use that for now to get this PR moving forward?